R/test.R

```diff
@@ -1,3 +1,27 @@
+#' Creates and returns run result data frame.
+#'
+#' @param file Name of the test runner file, a character vector of length 1.
+#' @param pass Whether or not the test passed, a logical vector of length 1.
+#' @param result Value (wrapped in a list) obtained by evaluating `file` or `NA`
+#'   if no value was obtained, such as with `shinytest`.
+#' @param error Error, if any, (and wrapped in a list) that was signaled during
+#'   evaluation of `file`.
+#'
+#' @return A 1-row data frame representing a single test run. `result` and
+#'   `error` are "list columns", or columns that may contain list elements.
+result_row <- function(file, pass, result, error) {
```

Comment:
If this function is not exported for public use, let's add a `#' @noRd` tag to prevent the man file from being produced.